### ISSUE TITLE
How do I get BL602_BOOT2_MINI to work and print the log?

### ISSUE THREAD

**Issue by balanceTWK** [bouffalolab/bl_iot_sdk#54]
![image](https://user-images.githubusercontent.com/30776697/119606695-981c7000-be25-11eb-96a7-2d9202961977.png)

![image](https://user-images.githubusercontent.com/30776697/119606591-74592a00-be25-11eb-92ba-eeed26f8d8e7.png)

**Comment by gamelaster** [comment]
The bl602_boot2_mini itself is the "Boot 2 Bin", so you need to set this .bin as Boot 2 Bin.

**Comment by balanceTWK** [comment]
> The bl602_boot2_mini itself is the "Boot 2 Bin", so you need to set this .bin as Boot 2 Bin.

Yes, I tried downloading the program like this. But there is no reaction and I am not sure if my BL602_BOOT2_MINI is running correctly.

**Comment by shchen-Lab** [comment]
@balanceTWK  

bl602_boot2_mini  is boot program,you need to set this .bin as Boot 2 Bin.  bl602_boot2_mini can't set Firmware Bin. you need set bl602_demo_event bin as Firmware Bin.
